javascript/javascript-quiz.md

```diff
@@ -1871,3 +1871,34 @@ console.log(cat.name);
 - [x] Wild
 - [ ] Tabby
 - [ ] Athena
+
+#### Q133. What will this code output to the log?
+
+```js
+var thing;
+let func = (str = 'no arg') => {console.log(str);}
+func(thing);
+func(null);
+```
+
+- [ ] null no arg
+- [ ] no arg no arg
+- [ ] null null
+- [ ] no arg null
```

Comment:
The answer `no arg null` Just execute the code in [Javascript Playground](https://playcode.io/)

Reply:
Added